appveyor.yml

```diff
@@ -48,6 +77,8 @@ environment:
 init:
   - cmake --version
   - msbuild /version
+  # set the default build configuration to Release
+  - if "%configuration%"=="" (set configuration="Release")
```

Comment:
Shouldn't we make the CMake build type explicit in the build matrix above? Just by browsing through the code, I saw the `Debug` entries and asked myself what build type is used if they are not explicitly mentioned.

Reply:
Done.